src/initialise_db.py

```diff
@@ -1,64 +1,73 @@
 '''
-Copyright (C) 2020 Link Shortener Authors
-Licensed under the MIT (Expat) License. See the LICENSE file found in the
-top-level directory of this distribution.
+Copyright (C) 2020 Link Shortener Authors (see AUTHORS in Documentation).
+Licensed under the MIT (Expat) License (see LICENSE in Documentation).
 '''
 from sanic import Blueprint
 
 from aiomysql import create_pool
 from aiomysql.sa import create_engine
 
-from sqlalchemy import MetaData, Table, Column, String, Integer, Boolean
+from sqlalchemy import MetaData, Table, Column, String
 from sqlalchemy.schema import CreateTable
 
 
 initdb_blueprint = Blueprint('intitialise_db')
 
 metadata = MetaData()
-initdb_blueprint.table = Table(
-    'links',
+initdb_blueprint.active_table = Table(
+    'active_links',
     metadata,
     Column('owner', String(50)),
+    Column('owner_id', String(50)),
```

Comment:
Google says that the ID can be 255 characters long: https://developers.google.com/identity/protocols/oauth2/openid-connect#obtainuserinfo
```suggestion
    Column('owner_id', String(255)),
```

Reply:
Fixed